### ISSUE TITLE
Error to send test emails.

### ISSUE THREAD

**Issue by RoRo-1** [WPManageNinja/fluent-smtp#70]
Good evening,
I am trying to send test emails using your plugin FluentSMTP (vers 2.1.0) but when I click on Email test - send I get the error you see in the image.
How can I send emails from your plugin?

Thank
![error-send-mailTest-FluentSMTP](https://user-images.githubusercontent.com/7474708/140547046-2a3fdd1e-000d-46c8-bd1a-076ac2968bb2.jpg)
s

**Comment by magnetoid** [comment]
Yes, this is something that is happening to me too, I am trying to connect to servers with credentials that i know are 100% accurate. And this credentials are working with other SMTP send plugins, but not with fluent SMTP

**Comment by deibertf** [comment]
Have you setup your port and protocols correctly ?
